clang/include/clang/Basic/BuiltinsPPC.def

```diff
@@ -35,6 +35,13 @@
 #define UNALIASED_CUSTOM_BUILTIN(ID, TYPES, ACCUMULATE, FEATURE)               \
   CUSTOM_BUILTIN(ID, ID, TYPES, ACCUMULATE, FEATURE)
 
+#define UNALIASED_CUSTOM_MMA_BUILTIN(ID, TYPES, FEATURE)                       \
```

Comment:
Can we document this new `define` to explain it for someone who is not familiar with the code?